src/frontend/utils/formatter.ts

```diff
@@ -73,3 +73,17 @@ export const sisteDatoIMnd = (måned: number, år: number): Date => {
 
 export const sorterFødselsdato = (fødselsDatoA: string, fødselsDatoB: string) =>
     familieDayjs(fødselsDatoA).isBefore(fødselsDatoB) ? 1 : -1;
+
+export const formaterTilKunFørstBokstavStor = (navn: string | undefined): string | undefined => {
```

Comment:
Is this for the name? We don't change the capitalization in the app, and should not do this here either.

Reply:
I agree, but Gunn is asking for this change. I think she has considerations from designer point of view